docs/conf.py

```diff
@@ -124,11 +124,14 @@ def setup(sphinx):
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 html_theme = 'sphinx_rtd_theme'
-
```

Comment:
I'd leave this line in. The file separates options with empty lines for readability.